pkg/config/model/types.go

```diff
@@ -67,6 +67,7 @@ type Reader interface {
 
 	// GetKnownKeys returns all the keys that meet at least one of these criteria:
 	// 1) have a default, 2) have an environment variable binded, 3) are an alias or 4) have been SetKnown()
+	// Note that it returns the keys lowercased.
```

Comment:
Did you consider renaming the function to something like `GetKnownLowerKeys`?

Reply:
As the name is the same as the underlying function from `viper` I didn't, but it would be a nice way to avoid unexpected behaviors for users, I'll update that !

Reply:
Also updated `GetAllKeys` to `GetAllKeysLowercased`, as it's the exact same behavior (`viper` converts all keys to lowercased)